src/main/java/graphql/schema/GraphQLScalarType.java

```diff
@@ -29,34 +32,34 @@
  * for example, a GraphQL system could define a scalar called Time which, while serialized as a string, promises to
  * conform to ISO‐8601. When querying a field of type Time, you can then rely on the ability to parse the result with an ISO‐8601 parser and use a client‐specific primitive for time.
  * <p>
- * From the spec : https://spec.graphql.org/October2021/#sec-Scalars
+ * From the spec : <a href="https://spec.graphql.org/October2021/#sec-Scalars">...</a>
  * </blockquote>
  * <p>
  * graphql-java ships with a set of predefined scalar types via {@link graphql.Scalars}
  *
  * @see graphql.Scalars
  */
 @PublicApi
-public class
-GraphQLScalarType implements GraphQLNamedInputType, GraphQLNamedOutputType, GraphQLUnmodifiedType, GraphQLNullableType, GraphQLDirectiveContainer {
+@NullMarked
+public class GraphQLScalarType implements GraphQLNamedInputType, GraphQLNamedOutputType, GraphQLUnmodifiedType, GraphQLNullableType, GraphQLDirectiveContainer {
 
     private final String name;
-    private final String description;
+    private final @Nullable String description;
     private final Coercing<?, ?> coercing;
     private final ScalarTypeDefinition definition;
     private final ImmutableList<ScalarTypeExtensionDefinition> extensionDefinitions;
     private final DirectivesUtil.DirectivesHolder directivesHolder;
-    private final String specifiedByUrl;
+    private final @Nullable String specifiedByUrl;
 
     @Internal
     private GraphQLScalarType(String name,
-                              String description,
+                              @Nullable String description,
```

Comment:
By definition in the specification, the description, and specified by URL can be null.